testing/InstallTestbed.ps1

```diff
@@ -0,0 +1,309 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [string]$ResourceGroup,
+
+    [string]$VMUsername = "admin.ackbar",
```

Comment:
$VMUsername is never used. Recommend either deleting unless you had intended to use it somewhere.
```suggestion
```